### PR TITLE
107 item details change status ready review

### PR DESCRIPTION
**Pull request by ivarbanovv**

### CODE REVIEW

File: lib/ios/constants/screens/item_details_screen.rb [hunk 1]
```diff
@@ -10,8 +10,6 @@ class ItemDetailsScreen < Base
   button(:cancel_button, 'accessibility_id~Cancel')
   text(:move_to_review_text, 'accessibility_id~Move to Review')
   text(:move_to_ready_text, 'accessibility_id~Move to Ready')
-
-  # confirmation dialog
-  button(:confirm_yes, 'accessibility_id~Yes')
+  button(:move_to_review_text, 'xpath~//XCUIElementTypeStaticText[@value="Move to Review"]')
```
Comment (ivodimitrov) [inline lib/ios/constants/screens/item_details_screen.rb:13]: Is that:  `text(:move_to_review_text, 'accessibility_id~Move to Review')`

File: spec/ios/direct_client/login_spec.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 describe '[Log in ] Direct client' do
 
   it 'logs in their account' do
-    on(LoginScreen).user_login
+    on(LoginScreen).user_login 'partner'
```
Comment (ivodimitrov) [inline spec/ios/direct_client/login_spec.rb:6]: Should log in  `direct client` instead of `partner`

File: lib/ios/constants/screens/base.rb [hunk 3]
```diff
@@ -6,12 +6,22 @@ def self.build_url(endpoint)
     return test_url + endpoint
   end
 
-  def item_attribute_visible?(element_attribute)
-    @driver.find_elements(:accessibility_id, element_attribute).size != 0
+  def item_attribute_visible?(item_attribute)
+    @driver.find_elements(:accessibility_id, item_attribute).size != 0
+  end
+
+  def find_item_by_attribute(item_attribute)
```
Comment (ivodimitrov) [inline lib/ios/constants/screens/base.rb:13]: Could `find_item_by_attribute(item_attribute)` have two parameters - locator type &  item_attribute ?

File: lib/ios/constants/screens/item_details_screen.rb (hunk 1 above)
Comment (ivarbanovv) [reply lib/ios/constants/screens/item_details_screen.rb:13]: its actually a mixture of both, but in the end its clickable so I decided - button

File: spec/ios/direct_client/login_spec.rb (hunk 2 above)
Comment (ivarbanovv) [reply spec/ios/direct_client/login_spec.rb:6]: yeah I forgot about the new structure, will fix that

File: lib/ios/constants/screens/base.rb (hunk 3 above)
Comment (ivarbanovv) [reply lib/ios/constants/screens/base.rb:13]: sure

**Pull request merged**